website/src/components/Chat/ChatConfigForm.tsx

```diff
@@ -104,6 +111,10 @@ export const ChatConfigForm = memo(function ChatConfigForm() {
   const selectedModel = getValues("model_config_name"); // have to use getValues to here to access latest value
   const selectedPlugins = getValues("plugins");
   const presets = modelInfos.find((model) => model.name === selectedModel)!.parameter_configs;
+  const [customInstructions, setCustomInstructions] = useState<CustomInstructionsType>({
+    user_profile: "",
+    user_response_instructions: "",
+  });
```

Comment:
This state can be handled by react-hook-form. You just need to wrap the `CustomInstructions` with the `Controller` component.

Reply:
Thanks, could you please provide a commit/change for that here if you have time and it's small?